ci/build_cpp.sh

```diff
@@ -16,9 +16,30 @@ rapids-logger "Begin cpp build"
 
 sccache --zero-stats
 
-# This calls mambabuild when boa is installed (as is the case in the CI images)
-RAPIDS_PACKAGE_VERSION=$(rapids-generate-version) rapids-conda-retry mambabuild conda/recipes/librmm
+RAPIDS_PACKAGE_VERSION=$(rapids-generate-version)
+export RAPIDS_PACKAGE_VERSION
+
+if rapids-is-release-build; then
+    RAPIDS_CHANNEL=rapidsai
+else
+    RAPIDS_CHANNEL=rapidsai-nightly
+fi
```

Comment:
Very minor nit: I foresee this propagating everywhere. Should we turn this into a GHA tool? We could even make it return an array of usable channels so that we also have a centralized place for removing the nvidia channel once we no longer need it.

Reply:
Yeah, we should.  I'll get that set up

Reply:
PR up in rapidsai/gha-tools#143